src/main/java/io/jenkins/plugins/prism/PrismTheme.java

```diff
@@ -8,7 +8,7 @@
  * @author Ullrich Hafner
  */
 public enum PrismTheme {
-    PRISM("Default (light and dark mode)", "prism.css"),
+    PRISM("Default (light and dark mode)", "default.css"),
```

Comment:
Can you remove the assertion in the Test:

```
                .extracting(PrismTheme::getFileName)
                .isEqualTo("prism.css")
```

It actually does not verify anything useful and can be removed.

Reply:
Done - thanks